zebra-utils/zcash-rpc-diff

```diff
@@ -119,6 +123,8 @@ EXIT_STATUS=$?
 
 if [ "$1" == "getaddressutxos" ]; then
     set "getaddressbalance" "$2"
+elif [ "$1" == "getrawmempool" ]; then
+    set "getrawmempool"
```

Comment:
This was a bit confusing to me when I saw the output, but I think it works ok for now. I just want to make a note in case someone else thinks the same thing.

```suggestion
    # Call `getrawmempool` again as a dummy request, the script isn't set up to do multiple cross-check calls
    set "getrawmempool"
```